queries/scala/highlights.scm

```diff
@@ -1,55 +1,62 @@
 ; CREDITS @stumash (stuart.mashaal@gmail.com)
 
+(class_definition
+  name: (identifier) @type.definition)
+
+(enum_definition
+  name: (identifier) @type.definition)
+
+(object_definition
+  name: (identifier) @type.definition)
+
+(trait_definition
+  name: (identifier) @type.definition)
+
+(full_enum_case
+  name: (identifier) @type.definition)
+
+(simple_enum_case
+  name: (identifier) @type.definition)
```

Comment:
no, they are all just `@type`. `@type.definition` is just a confusing capture that are meant for type aliases only (C's typedef)

Reply:
Gotcha, I updated them then.

Reply:
Should we consider renaming `@type.definition` to `@type.alias` to make it less confusing?

Reply:
yeah, that would be a lot better. Probably, even better when upstream to Neovim because this is how we got that name.

Reply:
Just a reminder: all upstream Neovim did was to add some default links to suitable-looking "standard groups"; these are not "blessed" in any way. Standardization of capture names is something only nvim-treesitter is interested in.